app/models/iiif_content_search_response.rb

```diff
@@ -56,6 +60,11 @@ def next_page?
     search.num_found >= (search.start + search.rows)
   end
 
+  def ignored_params
+    valid_iiif_params = %w[motivation date user]
```

Comment:
Should we pull `valid_iiif_params` up to the class-level or into the configuration?

Reply:
@cbeer How does this look? I moved the value to `settings.yml` and renamed it to `ignored_request_params`.